go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/avast/retry-go v3.0.0+incompatible
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
 	github.com/docker/go-connections v0.4.0
-	github.com/ethereum/go-ethereum v1.12.0
+	github.com/ethereum/go-ethereum v1.12.2
```

Comment:
I thought we had issues with this change due to the chainlink repo not having updated yet. Please very this change doesn't break e2e tests in core before we merge this.

Reply:
so much for them using semver?

Reply:
@tateexon good call, this field doesn't exist in `toml/config.go` of `go-ethereum` anymore:
```
txpool.DefaultConfig
```

I'll add that `geth` vulnerability to ignored ones and create a ticket in JIRA to remove it once `chainlink-core` [has bumped the dependency](https://github.com/smartcontractkit/chainlink/pull/10264). I don't like having vulnerability check always red.